examples/python-shell.py

```diff
@@ -0,0 +1,85 @@
+import asyncio
+import sys
+import shlex
+
+from yapapi.payload import vm
+from yapapi.services import Service
+
+from service_manager import ServiceManager
+
+
+class PythonShell(Service):
```

Comment:
While the previous `Clock` example is commented - this one is not so much.
I guess we're using the same trick as in galatea with running kinf-of server process on provider that handles 0mq messages? Is this explained anywhere?

Reply:
Provider code is in `examples/python-shell/provider`.
I don't think we should explain this in more detail. This is not the purpose of this example.